skbio/io/format/genbank.py

```diff
@@ -0,0 +1,865 @@
+same "Feature Table" layout in their plain text flat file formats.
+
+However, the header and sequence sections of an EMBL file are very
+different in layout to those produced by GenBank/DDBJ.
+
+Feature Table Documentation:
+http://www.insdc.org/files/feature_table.html
+ftp://ftp.ncbi.nih.gov/genbank/docs/FTv10_3.html
+
+The sequence in the ``'ORIGIN'`` section is always in lowercase for the
+GenBank files downloaded from NCBI. For the RNA molecules, ``'t'`` (thymine),
+instead of ``'u'`` (uracil) is used in the sequence. All GenBank writers
+follow these traditions while writing GenBank files.
+
+Format Parameters
+-----------------
+
+Reader-specific Parameters
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+The ``constructor`` parameter can be used with the ``Sequence`` generator
+to specify the in-memory type of each GenBank record that is parsed.
+``constructor`` should be ``Sequence`` or a sub-class of ``Sequence``.
+It is also detected by the unit label on the LOCUS line. For example, if it
+is ``'bp'``, it will be read into ``DNA``; if it is ``'aa'``, it will be read
+into ``Protein``. Otherwise, it will be read into ``Sequence``. This default
+behavior is overridden by setting ``constructor``.
+
+``lowercase`` is another parameter available for all GenBank readers.
+By default, it is set to ``True`` to read in the ``'ORIGIN'`` sequence
+as lowercase letters. This parameter is passed to ``Sequence`` or
+its sub-class constructor.
+
+``seq_num`` is a parameter used with the ``Sequence``, ``DNA``, ``RNA``, and
+``Protein`` GenBank readers. It specifies which GenBank record to read from the
+concatenated GenBank file with multiple records in it.
+
+Examples
+--------
+
+Reading and Writing GenBank Files
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Suppose we have the following GenBank file [example modified from [1_]::
+
+    LOCUS       3K1V_A       34 bp    RNA     linear   SYN 10-OCT-2012
+    DEFINITION  Chain A, Structure Of A Mutant Class-I Preq1.
+    ACCESSION   3K1V_A
+    VERSION     3K1V_A  GI:260656459
+    KEYWORDS    .
+    SOURCE      synthetic construct
+      ORGANISM  synthetic construct
+                other sequences; artificial sequences.
+    REFERENCE   1  (bases 1 to 34)
+      AUTHORS   Klein,D.J., Edwards,T.E. and Ferre-D'Amare,A.R.
+      TITLE     Cocrystal structure of a class I preQ1 riboswitch
+      JOURNAL   Nat. Struct. Mol. Biol. 16 (3), 343-344 (2009)
+       PUBMED   19234468
+    COMMENT     SEQRES.
+    FEATURES             Location/Qualifiers
+         source          1..34
+                         /organism="synthetic construct"
+                         /mol_type="other RNA"
+                         /db_xref="taxon:32630"
+    ORIGIN
+            1 agaggttcta gcacatccct ctataaaaaa ctaa
+    //
+
+>>> gb = ['LOCUS       3K1V_A     34 bp   RNA    linear   SYN 10-OCT-2012\\n',
+...       'DEFINITION  Chain A, Structure Of A Mutant Class-I Preq1.\\n',
+...       'ACCESSION   3K1V_A\\n',
```

Comment:
To clarify, adding `r` to the *module* string will fix that, I just noticed that I commented in the context of a list of strings...